src/internal_modules/roc_core/fast_random.cpp

```diff
@@ -7,6 +7,7 @@
  */
 
 #include "roc_core/fast_random.h"
+
 #include "roc_core/atomic_ops.h"
```

Comment:
Nit: we use single block for all roc includes.

Reply:
Yep, we don't use this style in this project. Just one block for all roc includes.

Reply:
ok, updated